framework/Supports/Socials_Feed.php

```diff
@@ -0,0 +1,384 @@
+<?php
+
+namespace JustCoded\WP\Framework\Supports;
+
+use JustCoded\WP\Framework\Admin\Theme_Settings;
+use Facebook\Facebook as FacebookSDK;
+use MetzWeb\Instagram\Instagram as InstagramSDK;
+
+
+/**
+ * Class SocialsFeed
+ *
+ * @package JustCoded\ThemeFramework\Supports
+ */
+class Socials_Feed {
+
+	/**
+	 * Posts array
+	 *
+	 * @var $fb_posts
+	 */
+	public $posts = array();
+
+
+	const POST_TYPE = 'socials_feed';
+	const TAXONOMY = 'socials_category';
+
+	/**
+	 * SocialsFeed constructor.
+	 */
+	public function __construct() {
+		add_action( 'init', array( $this, 'register_socials_posttype' ) );
+		add_action( 'init', array( $this, 'register_socials_taxonomy' ) );
+		add_action( 'init', array( $this, 'create_settings_panel' ) );
+
+		add_action( 'jtf_social_sheduler', array( $this, 'get_social_posts' ) );
+		if ( ! wp_next_scheduled( 'jtf_social_sheduler' ) ) {
+			wp_schedule_event( time(), 'hourly', 'jtf_social_sheduler' );
+		}
+
+		add_action( 'init', array( $this, 'insert_posts' ) );
+
+	}
+
+	/**
+	 * Register Socials post type
+	 */
+	public function register_socials_posttype() {
+		register_post_type( self::POST_TYPE, array(
+			'labels'       => array(
+				'name' => 'Socials Feed',
+			),
+			'public'       => true,
+			'hierarchical' => true,
+			'taxonomies'   => array(),
+		) );
```

Comment:
I think it should be hidden from admin interface. Don't know why you need "hierarchical" here?

And I think it will be useful to make visible in admin or not - a class option, so it can be configured easily in a theme